src/views/News/components/NewsItem/index.jsx

```diff
@@ -8,7 +16,15 @@ import { useState } from 'react';
 import { Link } from 'react-router-dom';
 import styles from './NewsItem.module.css';
 
-const NewsItem = ({ posting, unapproved, size, handleNewsItemEdit, handleNewsItemDelete }) => {
+const NewsItem = ({
+  posting,
+  unapproved,
```

Comment:
Since this is a boolean, can we name it with an `is` beforehand like `isUnapproved`? It's a personal preference, but I think it helps imply the boolean type for non-typescript files.